pkg/lang/ir/compile.go

```diff
@@ -173,14 +173,10 @@ func (g Graph) EnvString() []string {
 
 func (g Graph) DefaultCacheImporter() (*string, error) {
 	// The base remote cache should work for all languages.
-	v := version.GetGitTagFromVersion()
-	if v == "" {
-		v = "latest"
-	}
 	res := fmt.Sprintf(
 		"type=registry,ref=docker.io/%s/python-cache:envd-%s",
 		viper.GetString(flag.FlagDockerOrganization),
-		v)
+		version.GetGitTagFromVersion())
```

Comment:
Maybe we can rename the function.

Reply:
@zwpaper Any advice?

Reply:
IMHO, now it is this function's responsibility to retrieve the docker image's tag, then it may be appropriate to make that clearer from the name

Reply:
@Thrimbda Yes, and the question is which name will be better.
`GetImageTagFromVersion`?

Reply:
GetGitTag seems a internal implementation to define tag because the version is injected during building phase when `make` reads the git tag as the version. For a external use API, I think we shoud care about what the API is for finally.
It seems that this function give a short string as the version of a image tag. I think `GetVersionForImageTag` will be better.

Reply:
`GetVersionForImageTag` seems great to me

Reply:
GetVersionForImageTag LGTM

Reply:
PTAL